### ISSUE TITLE
[Components section] "Eyebrow texts are not read by screen readers" message implementation

### ISSUE THREAD

**Issue by JennySanchez** [carbon-design-system/carbon-for-ibm-dotcom-website#900]
🚨🚨🚨 **Make sure to confirm with @shixiedesign and @kennylam the accessibility fix is done before starting this work.** 🚨🚨🚨

### Problem to solve

(Per Shixie's converation with desigenrs in Slack)

"Screen reader to skip eyebrow text is intentionally added via [this PR](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/4870/files#diff-b90e80ea43e6f2127692daa38c6189f8e42ffd86cb42538e1a128ba8ffbac04eR30) in January. After a bit of research, I can agree that we should continue to skip reading eyebrow text. The primary reason for doing this is
1. we want to continue the anchor link wrap the whole card.
2. verbose screen reading is confusing.
So this is quite important IMO for content writing for anything card, esp. Feature card. We need to clearly state in our website documentation of card & feature card."

### Tasks

- [ ] Audit the components website pages we need to add the message to.
- [ ] Craft the message.
- [ ] Decide where we should add the message to on the website page.
- [ ] Add the message to the website pages.

### Acceptance criteria

- [ ] message added to the website pages from the audit
- [ ] Message can be seen live on the website

**Comment by JennySanchez** [comment]
@shixiedesign Can you add the list of the component website pages we need to add the message?

**Comment by shixiedesign** [comment]
It is still relevant, but dev hasn't resolved it yet. Please watch [this issue on Card](https://app.zenhub.com/workspaces/carbon-for-ibmcom-5d449f3642eb1962336cbe52/issues/carbon-design-system/carbon-for-ibm-dotcom/4613) as dev figure out a solution. I will add it as a blocker to starting this work.

**Comment by RobertaJHahn** [comment]
No longer needed.  Eyebrow text is now readable.  Closing
